src/resources/js/views/datepicker.js

```diff
@@ -282,6 +282,24 @@ tribe.events.views.datepicker = {};
 	obj.handleMousedown = function( event ) {
 		var $datepickerButton = event.data.target;
 		var state = $datepickerButton.data( 'tribeEventsState' );
+		var tapHide = false;
+
+		if ( 'touchstart' === event.type ) {
```

Comment:
2 notes here:
1. it would be good to separate the `touchstart` event handler from the `mousedown` handler
2. it would also be good to have a flag for this so that all this code below here doesn't run on every single tap, but only the first. especially for mobile, since their processors are smaller than computers, saving computing power is even more important.